go/protocols/materialize/client_transactor_test.go

```diff
@@ -1,380 +0,0 @@
-		require.NoError(t, err)
-		require.Nil(t, rpc.Flush())
-
-		transactor.startCommitErr = fmt.Errorf("mysterious start-commit failure")
-		_, _, err = rpc.StartCommit(&pf.Checkpoint{
-			Sources: map[pf.Journal]pc.Checkpoint_Source{"foobar": {ReadThrough: 123}}})
-		require.EqualError(t, err, "reading StartedCommit: transactor.StartCommit: mysterious start-commit failure")
-	})
-
-	t.Run("async-commit-error", func(t *testing.T) {
-		var (
-			transactor = &testTransactor{}
-			combiner   = &pf.MockCombiner{}
-		)
-		server.Transactor = transactor
-
-		var rpc, err = openTransactions(combiner, "{}")
-		require.NoError(t, err)
-		require.Nil(t, rpc.Flush())
-
-		transactor.commitOp = client.NewAsyncOperation()
-
-		_, opAcknowledged, err := rpc.StartCommit(&pf.Checkpoint{
-			Sources: map[pf.Journal]pc.Checkpoint_Source{"foobar": {ReadThrough: 123}}})
-		require.NoError(t, err)
-		require.NoError(t, rpc.Acknowledge())
-
-		// Race some document loads.
-		transactor.Loaded = map[int][]interface{}{0: {"found", "also-found"}}
-		require.NoError(t, rpc.AddDocument(0, tuple.Tuple{1}.Pack(), []byte("[1]"), json.RawMessage(`"one"`)))
-		require.NoError(t, rpc.AddDocument(1, tuple.Tuple{2}.Pack(), []byte("[2]"), json.RawMessage(`"one"`)))
-
-		transactor.commitOp.Resolve(fmt.Errorf("mysterious async commit failure"))
-		require.EqualError(t, opAcknowledged.Err(),
-			"reading Acknowledged: commit failed: mysterious async commit failure")
-
-		// Send a raced Load (after we know that `opAcknowledged` has resolved).
-		// While under the hood it sees EOF due to the stream break,
-		// expect it's mapped to a causal error for the user.
-		require.EqualError(t, rpc.AddDocument(0, tuple.Tuple{3}.Pack(), []byte("[3]"), json.RawMessage(`"one"`)),
-			"reading Acknowledged: commit failed: mysterious async commit failure")
-	})
-}
-
-// testServer implements DriverServer.
-type testServer struct {
-	group  *task.Group
-	server *server.Server
-
-	OpenRx   Request_Open
-	OpenedTx Response_Opened
-	Transactor
-}
-
-func newTestServer(t *testing.T) *testServer {
-	pb.RegisterGRPCDispatcher("local")
-
-	var instance = &testServer{
-		group:  task.NewGroup(pb.WithDispatchDefault(context.Background())),
-		server: server.MustLoopback(),
-	}
-	RegisterConnectorServer(instance.server.GRPCServer, instance)
-	instance.server.QueueTasks(instance.group)
-
-	t.Cleanup(func() {
-		instance.group.Cancel()
-		instance.server.BoundedGracefulStop()
-		require.NoError(t, instance.group.Wait())
-	})
-	instance.group.GoRun()
-
-	return instance
-}
-
-func (t *testServer) Client() ConnectorClient {
-	return NewConnectorClient(t.server.GRPCLoopback)
-}
-
-var _ ConnectorServer = &testServer{}
-
-func (t *testServer) Materialize(stream Connector_MaterializeServer) error {
-	var request, err = stream.Recv()
-	if err != nil {
-		return err
-	}
-	if request.Open == nil {
-		panic("not an open")
-	}
-
-	return RunTransactions(stream, *request.Open, t.OpenedTx, t.Transactor)
```

Comment:
Since we're deleting this test, it doesn't look like the materialization transactor go code is used anywhere in this repo, and we'd really have no reason not to move this to the connectors repo, unless I'm missing something?

Reply:
That's right. I didn't remove it yet because it's still referenced (though pinned) by connectors.